CrossCutting/Authorization/JwtTokenGenerator.cs

```diff
@@ -24,11 +25,16 @@ public class JwtTokenGenerator : IJwtTokenGenerator
         public string CreateJwtToken()
         {
             var now = DateTime.UtcNow;
+            var payload = new Dictionary<string, object>
+            {
+                { "isSU", true }
+            };
 
             var jwtToken = _tokenHandler.CreateToken(new SecurityTokenDescriptor
             {
                 Expires = now.AddDays(_jwtOptions.TokenLifeTime),
-                SigningCredentials = _signingCredentials
+                SigningCredentials = _signingCredentials,
+                Claims = payload
```

Comment:
It is the same that` jwtToken.Payload.Add("isSU", true); `?

Reply:
I do not know, @msimionati could you share two deserialized tokens generated in these ways?

Reply:
Apparently it's the same:

This is how it is generated as it is now.
HEADER:ALGORITHM & TOKEN TYPE
{
  "alg": "RS256",
  "typ": "JWT"
}
PAYLOAD:DATA
{
  "isSU": true,
  "nbf": 1605191023,
  "exp": 1605277420,
  "iat": 1605191023
}

This is how it is generated as a question @nreal (jwtToken.Payload.Add("isSU", true);)
HEADER:ALGORITHM & TOKEN TYPE
{
  "alg": "RS256",
  "typ": "JWT"
}
PAYLOAD:DATA
{
  "nbf": 1605190803,
  "exp": 1605277202,
  "iat": 1605190803,
  "isSU": true
}

Reply:
Good